README.md

```diff
@@ -200,9 +200,10 @@ Follow the [quick start Jupyter Notebook](./feathr_project/feathrcli/data/feathr
 - [x] Private Preview release
 - [x] Public Preview release
 - [ ] Future release
```

Comment:
let's add what we already supports here so users can easily to find them?

like point-in-time join, aggregation, offline transformation, preprocessing?

Reply:
Maybe in a separate PR? we should also add Feathr highlights